### PR TITLE
Updated several casks: mkvtools, packet-peeper / Added Deploymate, Panic ShrinkIt

### PR DESCRIPTION
**Pull request by winkelsdorf**
Edit: See discussion below

### CODE REVIEW

**Comment by winkelsdorf** [comment]
Filezilla & MacID can be omitted, in the meantime another PR was accepted including them..

**Comment by vitorgalvao** [comment]
Could you please fix and rebase?

**Comment by winkelsdorf** [comment]
Done :)

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by winkelsdorf** [comment]
Sure, done.

File: Casks/deploymate.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask :v1 => 'deploymate' do
+  version '1.2.8'
+  url "https://s3.amazonaws.com/deploymate/download/Deploymate#{version.gsub('.', '')}.zip"
```
Comment (vitorgalvao) [inline Casks/deploymate.rb]: Needs a `# amazonaws.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5)).

**Comment by vitorgalvao** [comment]
Just noticed you have a weird stanza order; you should follow the other casks:

```ruby
version
sha256

url
…
```

**Comment by winkelsdorf** [comment]
Pushing & Squashing reordered Casks in a short moment.

Regarding the Amazon AWS: See http://www.deploymateapp.com/, that's the official link. The only other possibility is follow the meta refresh html tag in http://www.deploymateapp.com/download/latest/. That's iirc not possible with Casks.

**Comment by winkelsdorf** [comment]
Done, should be correct now.

File: Casks/deploymate.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+cask :v1 => 'deploymate' do
+  version '1.2.8'
+  sha256 '7e990c4ce807510ccfed544b968f049c28aec6a7988d068b5412fc93466b79f6'
+
+  # amazonaws.com is the official download host per the vendor homepage
+  url "https://s3.amazonaws.com/deploymate/download/Deploymate#{version.gsub('.', '')}.zip"
+
```
Comment (vitorgalvao) [inline Casks/deploymate.rb]: This empty line shouldn’t be here.

File: Casks/shrinkit.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'shrinkit' do
+  version '1.3.2'
+  sha256 'c076168a05d8a664cfac9bdee2f7a5f7cb317ade7d9f00c2cd0d49b33294a943'
+  
+  url "https://download.panic.com/shrinkit/ShrinkIt%20#{version}.zip"
+
```
Comment (vitorgalvao) [inline Casks/shrinkit.rb]: Same here.

**Comment by vitorgalvao** [comment]
Looks like some trailing whitespace got through, in shrinkit, causing the build to fail.

**Comment by winkelsdorf** [comment]
Hmm, ok. Going to create & squash a new commit.

Isn't travis a bit 'strict'? It's 1) valid ruby and 2) following the other files:

I used 1Password from Main Casks as template:
https://github.com/caskroom/homebrew-cask/blob/master/Casks/1password.rb

See the New-Line in between.. 25 days ago someone accepted a PR with a single newline. Anyway, if I quick browser through the official repo, I see a lot of mistakes, like double Name in 0xdbe-eap.rb, space in 1Password, double name in 8tracksradiohelper.rb and so on, just for the first 10 files.

Looks like Cake is a bit nit-picky, isn't it ;)

Anyway, I create a last commit. Probably better I leave the Casks to others..

**Comment by vitorgalvao** [comment]
> Isn't travis a bit 'strict'?

Those are custom rules, to catch more cases.

> I used 1Password from Main Casks as template

That’s an atypical cask, with conditionals and differently names apps depending on the version.

> a lot of mistakes, like double Name

That is not a mistake; multiple `name` stanzas are allowed, when the app has multiple canonical names.

> Looks like Cake is a bit nit-picky, isn't it ;)

Consistency is important. When you manage a project with this amount of files and PRs, structure is imperative. By not following carefully though out rules — yes, even in the order of stanzas and the addition of whitespace, you’re making casks harder to parse, and my job in managing them that much harder. This is incredibly time-consuming; details matter.

**Comment by winkelsdorf** [comment]
Ok, I see. From that perspective it's really better to follow the rules strictly, I will take care of when doing new commits.

Travis CI passed now. Have a look at it and merge the PR if you like :)

**Comment by vitorgalvao** [comment]
Looks good. Thank you for your patience on this.

**Comment by winkelsdorf** [comment]
You're welcome! Thank you for your patience, too and for pointing out the needs and best practices :)

**Pull request merged**